molecule/default/molecule.yml

```diff
@@ -33,6 +33,11 @@ platforms:
       - icinga2client
       - icinga2stdclient
     image: debian:9
+  - name: debian-icinga2-std-client8
+    groups:
+      - icinga2client
+      - icinga2stdclient
+    image: debian:8
 provisioner:
```

Comment:
The scenario is already running for ~15 minutes and sometimes fails due to travis issues. So I would like to avoid adding more legacy OS versions in here.
Maybe we can create a another scenario that just installs the client and monitoring plugins where we add some of the older OSes?

Reply:
Agreed.